src/snapred/ui/workflow/ReductionWorkflow.py

```diff
@@ -50,7 +53,7 @@ def __init__(self, parent=None):
                 "Reduction",
                 continueAnywayHandler=self._continueAnywayHandler,
             )
-            .addNode(self._nothing, self._reductionSaveView, "Save")
+            # .addNode(self._nothing, self._reductionSaveView, "Save")
```

Comment:
I would delete this line entirely, or add a comment that this will be cleaned up later

Reply:
done, also updated the completion text